suricata.yaml.in

```diff
@@ -975,10 +1004,13 @@ app-layer:
            double-decode-path: no
            double-decode-query: no
 
+           # Can disable LZMA decompression
            #lzma-enabled: yes
-           # LZMA decompression memory limit.
+           # Memory limit usage for LZMA decompression dictionary
+           # Data is decompressed until dictionary reaches this size
            #lzma-memlimit: 1 Mb
-           # Compression bomb output limit.
+           # Maximum decompressed size with a compression ratio
+           # above 2048 (only reachable by LZMA)
```

Comment:
"only reachable by LZMA" took me a few reads to understand. Perhaps update to "only LZMA can reach this ratio" ?

Reply:
ok